docs/servers/server-software.md

```diff
@@ -23,6 +23,7 @@ mentions:
     - wu-vincent
     - smartcmd
     - Adrian8115
+    - SculkTeams
```

Comment:
Mentions should be GitHub usernames, "SculkTeams" doesn't seem to exist.

Reply:
I just made the change, I did not know, thank you